packages.toml

```diff
@@ -0,0 +1,11 @@
+[dash]
```

Comment:
This file should be placed in the .gitignore, as we are only putting the server on this repository and not the packages that it holds.